src/commands/env/alias/set.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { AnyJson } from '@salesforce/ts-types';
+import { Flags } from '@oclif/core';
+
+import SfCommand from '../../../sf-command';
+
+export default class EnvAliasSet extends SfCommand {
+  public static description = `set env alias
+
+  Set an alias for an environment.
+  `;
+
+  public static examples = ['sf env alias set [alias] -t [env]'];
+
+  public static args = [{ name: 'alias' }];
+
+  public static flags = {
+    targetEnv: Flags.string({
+      char: 't',
+      description: 'The environment to set an alias for',
+    }),
+  };
+
+  public async run(): Promise<AnyJson> {
+    const { flags, args } = await this.parse(EnvAliasSet);
+
+    this.log(`Setting ${args.alias as string} as ${flags.targetEnv}...\n`);
```

Comment:
You could also call `this.aliases...` to actually set an alias so it would show up in some of the other interactive commands.

Reply:
Ditto with all the other alias commands.

Reply:
Wouldn't this have to be `environment.aliases` instead of `this.aliases`?

Reply:
Maybe I'm not understanding this at all but I pushed up a WIP to `re/env_alias` that is exhibiting some strange behavior and not finding environments as it should: https://github.com/salesforcecli/cli-taxonomy-experiement/blob/re/env_alias/src/commands/env/alias/set.ts#L32-L49
![image](https://user-images.githubusercontent.com/1084688/115750240-f60cf200-a365-11eb-8d2b-db28a7546e10.png)